pyquil/api/_quantum_processors.py

```diff
@@ -0,0 +1,29 @@
+##############################################################################
+# Copyright 2018 Rigetti Computing
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+
+from pyquil.api import Client
+from pyquil.api._client import _parse_error
+from pyquil.device import Device
+
+
+# TODO(andrew): update to get new ISA and instantiate a device, using new endpoints (and client functions)
+def get_device(client: Client, processor_id: str) -> Device:
```

Comment:
Note: integration point between @erichulburd and I. May remain a stub for this PR, depending on logistics. (Note that this PR targets merging to an issue branch, not the v3 branch, so it should be fine either way)